documentation/source/theory/rmg/dynamics.rst

```diff
@@ -0,0 +1,96 @@
+.. _dynamics:
+
+*******************
+Dynamics Criterion
+*******************
+
+When dealing with more complex chemical mechanisms the standard RMG flux criterion
+has trouble picking up key chain branching reactions and has limited guarantees
+that it accurately represents the concentrations of all species.  The dynamics
+criterion is a measure of how much a given reaction affects core concentrations.
+This allows it to pick up key low flux chain branching reactions and better represent
+species concentrations.
+
+Calculating the Dynamics Criterion
+==================================
+Let us define production and consumption rates for a given species
+:math:`\frac{dc_i}{dt} = P_i(t) - L_i(t)`
+
+Let us define a dimensionless concentration variable we will refer to as the
+accumulation number Ac for a given species
+
+:math:`Ac_{spc,i} = \frac{P_i}{L_i} \approx \frac{\bar{c_{i}}}{c_{i0}}`
+
+where :math:`\bar{c_i}` is the steady state concentration or more specifically the
+concentration at which :math:`P_i = L_i` assuming :math:`L_i` scales with :math:`c_i`
+and :math:`c_{i0}` is the current concentration.
+
+This species accumulation number is a measure of how far species i is from steady state.
+
+Since this number can only be calculated for core species, by itself it is only a
+measure of the behavior of species i within the reaction network.
+
+However if we consider models with and without some edge reaction j we can define
+
+:math:`\Pi_{Ac,i,j} = \frac{Ac_{spc,i,withj}}{Ac_{spc,i,withoutj}}`
+
+Which is a measure of how much the concentration of species i is impacted by
+reaction j.
+
+In order to directly compare multiple reactions we can then sum over all
+core species involved in reaction j to get our criterion the dynamics number.
+
+:math:`\sum_{i\in core} |Ln(\Pi_{Ac,i,j})| = Dy > \epsilon`
+
+Surface Algorithm
+=================
+One common issue with the dynamics criterion is that as can be seen in our theory
+section, it treats all core species equally.  Because of this, if the dynamics
+criterion is set too low it enters a feedback loop where it adds species and then
```

Comment:
This is nit-picky, but I think you should add a coma here. 
"where it adds species, and then ..."